client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -16,22 +16,25 @@ import Gridicon from 'gridicons';
  * Internal dependencies
  */
 import Animate from 'components/animate';
-import ProgressIndicator from 'components/progress-indicator';
 import Button from 'components/button';
-import analytics from 'lib/analytics';
 import { recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
 import QuerySiteConnectionStatus from 'components/data/query-site-connection-status';
-import { canCurrentUser } from 'state/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
 import { getUpdatesBySiteId } from 'state/sites/updates/selectors';
-import { updateWordPress } from 'state/sites/updates/actions';
 import {
+	canCurrentUser,
 	getSiteConnectionStatus,
 	isRequestingSiteConnectionStatus,
 	isSiteAutomatedTransfer,
-	isWordpressUpdateSuccessful,
 } from 'state/selectors';
 
+const WPAdminLink = ( { href, onClick, children } ) => (
+	<a href={ href } onClick={ onClick } target="_blank" rel="noopener noreferrer">
+		{ children }
+		<Gridicon icon="external" size={ 12 } />
+	</a>
```

Comment:
Let's use an external link component https://wpcalypso.wordpress.com/devdocs/design/external-link

Reply:
Why didn't I find this component when I was looking for it? 🤦‍♂️ Migrated the code to `ExternalLink`.